### ISSUE TITLE
Document relation to usethis

### ISSUE THREAD

**Issue by krlmlr** [cynkra/fledge#107]
Is fledge a complement? replacement?

E.g. what does it mean if you use `usethis::use_release_issue()`?

**Comment by maelle** [comment]
Notes for later. When using fledge one would _not_ use `usethis::use_version()` not `usethis::use_dev_version()`. Other than that the checklist from `usethis::use_release_issue()` is valid.

**Comment by maelle** [comment]
I feel this should be in the 0.1.0 milestone because the demo vignette recommends using `usethis::use_release_issue()`.

**Comment by krlmlr** [comment]
I no longer think the demo vignette should recommend that, with #27. Let's remove the note on `use_release_issue()` and focus later on documenting fledge's release process.

**Comment by maelle** [comment]
In the release automation, I am missing the very precise usethis checklist from `usethis::use_release_issue()`, as well as the use of an issue for box checking (it's easier than in cran-comments.md IMHO).

**Comment by krlmlr** [comment]
I agree that adoption will be easier if we market fledge as a more or less "natural" extension of usethis workflows.

**Comment by maelle** [comment]
Also state that fledge can be used _without_ usethis as well.

**Comment by maelle** [comment]
(well without usethis "explicit usage" :sweat_smile: )

**Comment by maelle** [comment]
I think this is still a missing piece. Or maybe with configuration (😅 ) fledge could support different preferences

- something closer to usethis workflows, a complement to the usethis checklist as opposed to a replacement.
- your workflow.
- releases to other repositories, for instance multiverse? https://r-multiverse.org/

**Comment by maelle** [comment]
Do you do that for igraph? https://github.com/igraph/rigraph/pull/2096/changes
